app/test_main.py

```diff
@@ -1,3 +1,23 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_human_ages",
+    [
+        (14, 14, [0, 0]),
+        (23, 23, [1, 1]),
+        (26, 0, [2, 0]),
+        (0, 29, [0, 3]),
+        (33, 35, [4, 4]),
+        (40, 0, [6, 0]),
+        (0, 45, [0, 6]),
+        (48, 50, [8, 7]),
+    ]
+)
+def test_get_human_age(
+        cat_age: int,
+        dog_age: int,
+        expected_human_ages: list
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected_human_ages
```

Comment:
Make sure you've tests checks all edged situations such as:

* if output should change with some integer value you should check that it is not changed with previous value;
* if function resive data out of normal range, such as negative numbers, zero, or realy large numbers;
* if the function receives an incorrect type of data it raises the correct exception.